pkg/taskqueue/queue.go

```diff
@@ -0,0 +1,37 @@
+package taskqueue
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/webhookx-io/webhookx/utils"
+	"time"
+)
+
+type TaskMessage struct {
+	ID string
+
+	data []byte
```

Comment:
consider msg pack to enhance data efficiency.

Reply:
Good point. Using Gob or any other binary serializers can improve the ingestion and consumption performance.

Reply:
We can optimize it on another PR.